index.html

```diff
@@ -5,6 +5,11 @@
 		<link rel="icon" type="image/svg+xml" href="/vite.svg" />
 		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
 		<title>Memory Game</title>
+		<link
```

Comment:
Widziałem że mamy w package.json tą paczke `toastify` dodaną, jeśli ona działa to tutaj usuwamy ten link?

Reply:
A faktycznie. Miałem problem z tym pobieraniem paczek przez npm i użyłem CDN, którego potem nie wykasowałem po naprawieniu tego problemu.